cpp/server/handler.h

```diff
@@ -37,7 +37,7 @@ class HttpHandler {
               const ReadOnlyDatabase<LoggedEntry>* db,
               const ClusterStateController<LoggedEntry>* controller,
               ThreadPool* pool, libevent::Base* event_base);
-  ~HttpHandler();
+  virtual ~HttpHandler();
```

Comment:
Unrelated! But I'll let it slip... this time! :wink: 

Did you get a warning for it?